docs/site-example/site.conf

```diff
@@ -209,8 +209,13 @@
     -- },
     -- define if the contact field is obligatory (optional)
     -- owner = {
-      -- obligatory = true
+      -- obligatory = true,
```

Comment:
Style of comments should be unified to. (see `roles`)

Reply:
is correct too